.babelrc

```diff
@@ -1,11 +1,20 @@
 {
 	"presets": [
-		"es2015",
+		[ "env", {
+			"targets": {
+				"browsers": [
+					"last 2 versions",
+					"Safari >= 9",
+					"iOS >= 9",
+					"not ie <= 10",
+					"> 1%"
+				]
+			}
+		} ],
 		"stage-2"
 	],
 	"plugins": [
 		"transform-runtime",
-		"transform-class-properties",
```

Comment:
`babel-preset-react` includes `babel-preset-flow`, which we probably don't need. We could probably use `babel-preset-react` still, but it doesn't seem necessary.